plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -5378,523 +5373,6 @@ public void testBucketedSelect()
         }
     }
 
-    @Test
-    public void testGroupedExecution()
-    {
-        try {
-            assertUpdate(
-                    "CREATE TABLE test_grouped_join1\n" +
-                            "WITH (bucket_count = 13, bucketed_by = ARRAY['key1']) AS\n" +
-                            "SELECT orderkey key1, comment value1 FROM orders",
-                    15000);
-            assertUpdate(
-                    "CREATE TABLE test_grouped_join2\n" +
-                            "WITH (bucket_count = 13, bucketed_by = ARRAY['key2']) AS\n" +
-                            "SELECT orderkey key2, comment value2 FROM orders",
-                    15000);
-            assertUpdate(
-                    "CREATE TABLE test_grouped_join3\n" +
-                            "WITH (bucket_count = 13, bucketed_by = ARRAY['key3']) AS\n" +
-                            "SELECT orderkey key3, comment value3 FROM orders",
-                    15000);
-            assertUpdate(
-                    "CREATE TABLE test_grouped_join4\n" +
-                            "WITH (bucket_count = 13, bucketed_by = ARRAY['key4_bucket']) AS\n" +
-                            "SELECT orderkey key4_bucket, orderkey key4_non_bucket, comment value4 FROM orders",
-                    15000);
-            assertUpdate(
-                    "CREATE TABLE test_grouped_joinN AS\n" +
-                            "SELECT orderkey keyN, comment valueN FROM orders",
-                    15000);
-            assertUpdate(
-                    "CREATE TABLE test_grouped_joinDual\n" +
-                            "WITH (bucket_count = 13, bucketed_by = ARRAY['keyD']) AS\n" +
-                            "SELECT orderkey keyD, comment valueD FROM orders CROSS JOIN UNNEST(repeat(NULL, 2))",
-                    30000);
-            assertUpdate(
-                    "CREATE TABLE test_grouped_window\n" +
-                            "WITH (bucket_count = 5, bucketed_by = ARRAY['key']) AS\n" +
-                            "SELECT custkey key, orderkey value FROM orders WHERE custkey <= 5 ORDER BY orderkey LIMIT 10",
-                    10);
-
-            // NOT grouped execution; default
-            Session notColocated = Session.builder(getSession())
```

Comment:
would that make sense to leave test assertion for `notColocated`? Or is that covered by other tests. I would assume the latter, but can you check please?

Reply:
Maybe. I did not found tests that strictly mimic this one. The question remains about the name of the test since it has nothing to do with grouped execution anymore